apps/andi/lib/andi_web/live/edit_organization_live_view.ex

```diff
@@ -225,22 +226,33 @@ defmodule AndiWeb.EditOrganizationLiveView do
     {:noreply, redirect(socket, to: "/organizations/#{socket.assigns.org.id}")}
   end
 
-  def handle_event("order-by", _, socket) do
+  def handle_event("order-by", %{"field" => field}, socket) do
     order_dir =
       case socket.assigns.order do
-        "asc" -> "desc"
+        %{^field => "asc"} -> "desc"
         _ -> "asc"
       end
 
-    sorted_datasets = socket.assigns.harvested_datasets |> Enum.reverse()
-    {:noreply, assign(socket, harvested_datasets: sorted_datasets, order: order_dir)}
+    sorted_datasets = sort_by_dir(socket.assigns.harvested_datasets, field, order_dir)
```

Comment:
I'd love to see this moved back into ecto query as well, but I digress...

Reply:
Yeah there is some opportunities to make another macro for table live view pages that I will probably add soon, so I can do that then